bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/i18n/fineoffsetweatherstation.properties

```diff
@@ -30,6 +30,8 @@ channel-type.fineoffsetweatherstation.battery-voltage.label = Battery Voltage
 channel-type.fineoffsetweatherstation.battery-voltage.description = The voltage of the battery
 channel-type.fineoffsetweatherstation.co2.label = CO₂
 channel-type.fineoffsetweatherstation.co2.description = Air Quality Indicator
+channel-type.fineoffsetweatherstation.free-stack-size.description = The available stack size. If it is reducing, it means the stack is using up.
+channel-type.fineoffsetweatherstation.free-stack-size.label = Free Stack Size
```

Comment:
It's documented as "Heap free", is it called "Stack" here on purpose?

Reply:
The screenshot here seems to confirm some inconsistency regarding stack vs. heap (label vs. channel id): https://community.openhab.org/t/fine-offset-weather-station-binding-discussion/134167/269